docs/DeveloperGuide.md

```diff
@@ -2,37 +2,207 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+Certain aspects of YMFC's code was inspired from the following member(s) IPs:
+
+[Sanjith](http://github.com/KSanjith/ip)
 
 ## Design & implementation
```

Comment:
could seperate the design and implementation parts, under design u include th class diagrams and under implementation you have the sequence diagrams